MetaPackage.md

```diff
@@ -0,0 +1,137 @@
+# Metapackages in Chocolatey
+
+Chocolatey has the concept of a "meta" package. These types of packages don't typically include any sort of logic that would leverage installation, upgrade, and uninstallation scripts. Alternatively, they provide a means to bundle _existing_ packages as dependencies, giving you a single package to install, which in turn will install all of those subsequent dependent packages.
+
+Controlling what packages you wish to include is handled entirely in the dependencies section of a nuspec file. In the following sections, we will work through the creation of a meta package.
+
+## Creating your metapackage
+
+Open up an administrative Powershell window and issue the following commands:
+
+```powershell
+
+#configure a location for your metapackage
+mkdir C:\packages
+
+#enter the newly created directory
+Set-Location C:\packages
+
+#Create a new package
+choco new -n metapackage

```

Comment:
```suggestion
choco new --name metapackage
```

I don't see a `-n` in the help output for the choco new command.  Is that an oversight in the docs?